dataactbroker/handlers/fileHandler.py

```diff
@@ -297,8 +301,16 @@ def getStatus(self):
             for jobId in jobs:
                 jobInfo = {}
                 jobType = self.jobManager.getJobType(jobId)
+
                 if jobType != "csv_record_validation" and jobType != "validation":
                     continue
+
+                # TODO Skip D1 file until validation is added, remove these lines once D1 validation is added
```

Comment:
Is there a way to make this more prominent? Maybe make a separate issue if one doesn't already exist? I don't think anyone's going to see this comment if they aren't specifically looking for it.